packages/block/karma.conf.js

```diff
@@ -6,8 +6,9 @@ module.exports = function(config) {
     preprocessors: {
       './test-build/**/*.js': ['browserify'],
     },
+    concurrency: 1,
```

Comment:
The simplest solution to an annoying problem :)

Reply:
Can you update this to a merge-ready PR? 😄

Reply:
Sure! doing it now.

Was waiting for the approval of you all, so I can clean the git tree and push it forward.